scripts/module/teleporter.nut

```diff
@@ -21,11 +21,16 @@ function teleport(player, location) {
   return false;
 }
 
-
 function onPlayerCommand( player, cmd, text ) {
   cmd = cmd.tolower();
   if(cmd == "t" || cmd == "teleport") {
     if(text) {
+      local coords = split(text, " ");
+      if (coords.len == 3 && player.Admin) {
+        local pos = Vector(coords[0].tofloat(), coords[1].tofloat(), coords[2].tofloat());
+        TeleportPlayer(player, pos, 0);
+        return 1;
+      }
```

Comment:
nit: I'd surround this block with empty lines to make sure that people can see it's independent

Reply:
Better? 🙂